pushed_authorize_request_handler.go

```diff
@@ -0,0 +1,63 @@
+package fosite
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/ory/fosite/i18n"
+	"github.com/ory/x/errorsx"
+)
+
+// NewPushedAuthorizeRequest validates the request and produces an AuthorizeRequester object that can be stored
+func (f *Fosite) NewPushedAuthorizeRequest(ctx context.Context, r *http.Request) (AuthorizeRequester, error) {
+	request := NewAuthorizeRequest()
+	request.Request.Lang = i18n.GetLangFromRequest(f.Config.GetMessageCatalog(ctx), r)
+
+	if r.Method != "" && r.Method != "POST" {
```

Comment:
When would r.Method be empty? Can we remove this check?

Reply:
This comes into play if you use one of the test frameworks. I guess I could remove it but I know I added it in my own implementation for some reason, but can't remember at the moment.

Reply:
Thanks, i removed it and resolved the failing test cases :)

https://github.com/ory/fosite/pull/660/commits/49729feaddcd3faaf43b1d06b601ac93b47599df

Reply:
@aeneasr Thanks.